develop/smart-contracts/evm/parachain-contracts.md

```diff
@@ -1,4 +1,70 @@
 ---
 title: Parachain Contracts
 description: Learn how Polkadot parachains such as Moonbeam, Astar, Acala, and Manta leverage the Ethereum Virtual Machine (EVM) and integrate it into their parachains.
----
\ No newline at end of file
+---
+
+# Parachain Contracts
+
+## Introduction
+
+One key factor underpinning Ethereum's growth is the ease of deploying to the EVM. The EVM, or Ethereum Virtual Machine, provides developers with a consistent and predictable execution environment for smart contracts. While the EVM is not perfect, its popularity and ease of deployment have far outweighed any shortcomings and resulted in the massive growth of EVM-compatible smart contract platforms. 
+
+Also integral to the proliferation of EVM-based smart contract networks is smart contract portability. Developers can take their smart contracts that they've deployed to Ethereum, and in many cases, deploy them to other EVM-compatible networks with minimal changes. More than "Copy/Paste" deployments, this enables chains' interoperability. Building a cross-chain application is much easier when both chains offer similar EVM compatibility. 
+
+## Why Adopt the EVM as a Polkadot Parachain?
```

Comment:
I would merge this a bit with "## EVM + The Power of Polkadot"

Moreover, I think this section needs a bit more images or a mermaid diagram. Maybe the classic "User -> Ethereum API (Precompiles) -> Subtrate API?" Wdyt?